### DISCUSSION TITLE
.dev.vars doesn't work in the v7 cloudflare examples...

### DISCUSSION THREAD

**Discussion by alindsay55661** [remix-run/react-router#12369] [Q&A]
I got my site up and running with v7, but I can't figure out how to get wrangler to pick up my `.dev.vars` file which it did perfectly with the previous `remix vite:dev` version... How can I inject secrets into the cloudflare development environment?

**Comment by alindsay55661** [comment]
I ended up doing this in my vite config... not sure if this is the "proper" way to inject secrets in dev mode...

```ts
vitePluginViteNodeMiniflare({
      entry: './workers/app.ts',
      miniflareOptions: (options) => {
        options.compatibilityDate = '2024-11-18'
        options.compatibilityFlags = ['nodejs_compat']
        options.bindings = {
          ...options.bindings,
          ...envBindings, // an object containing env secrets (read from process.env)
        }
      },
    }),
```

**Comment by mw10013** [comment]
This PR for the v7 cloudflare example may be helpful: Modify to use cloudflareDevProxy and load contents of wrangler.toml  
 https://github.com/remix-run/react-router-templates/pull/6

**Reply by alindsay55661** [reply]
This approach is certainly more familiar to the Remix way. Thank you for sharing!
